apps/src/gamelab/GameLab.js

```diff
@@ -36,6 +41,13 @@ var GameLab = function () {
   this.tickIntervalId = 0;
   this.tickCount = 0;
 
+  /**
+   * Redux Store holding application state, transformable by actions.
+   * @private {Store}
+   * @see http://redux.js.org/docs/basics/Store.html
+   */
+  this.reduxStore_ = createStore(gamelabReducer);
```

Comment:
Curious—design-wise is it the case that applab & gamelab's `AppName.store`s might move upwards to `StudioApp.prototype` or somewhere else shared at some point in the future?